.github/workflows/ci.yml

```diff
@@ -37,10 +37,12 @@ jobs:
         php: ['7.3', '7.4', '8.0', '8.1', '8.2', '8.3']
         min_stability: ['']
         name_suffix: ['']
+        phpunit_configuration: ['phpunit.xml.dist']
         include:
           - php: '8.3'
             min_stability: 'dev'
             name_suffix: ' (dev deps)'
+            phpunit_configuration: 'phpunit.10.5+.xml.dist'
```

Comment:
This is not the right approach IMO, as more jobs will start using PHPUnit 11 as soon as Prophecy gets a stable release.

What is the thing triggering a failure exit code with the old config ? Is it the deprecation about the config format or the warning related to the coverage ? If it's the later, I would suggest removing `--coverage-text` from the CI job (when we want code coverage, we can generate it locally, and when the CI is green, we generally don't go look at the output)